File: pinyougou-manager-web/src/main/webapp/js/service/sellerService.js
app.service("sellerService",function ($http) {

    this.search = function (page,size,sellerSearch) {
        return $http.post("../seller/search.do?page="+page+"&size="+size,sellerSearch);
    }

    this.findSellerById = function (sellerId) {
        return $http.get("../seller/findSellerById.do?sellerId="+sellerId);
    }

    this.updateStatus = function (sellerId, status) {
        return $http.get("../seller/updateStatus.do?sellerId=" + sellerId + "&status=" + status);
    }
})

File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/SellerService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbSeller;
import entity.PageResult;

/**
 * 商家接口
 */
public interface SellerService {

    public void add(TbSeller tbSeller);

    public PageResult search(TbSeller tbSeller, int pageNum, int pageSize);

    public TbSeller findSellerById(String id);

    public void updateStatus(String sellerId,String status);
}
